RawMediaAccess/.idea/misc.xml

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="DesignSurface">
+    <option name="filePathToZoomLevelMap">
+      <map>
+        <entry key="..\:/Users/Usuario/Downloads/Microsoft/Repos/communication-services-android-quickstarts/RawMediaAccess/app/src/main/res/drawable/acs_spool_ui_blue_border.xml" value="0.3728205128205128" />
```

Comment:
I think we'll need to update this path so it's relative and doesn't have your username in it

Reply:
do we need these files on ` .idea`  folder? isn't build.gradle enough?

Reply:
this change coulkd be caused by a copy paste file let see what happens when i remove this